packaging/nnstreamer.spec

```diff
@@ -117,6 +121,18 @@ BuildRequires: lcov
 %if 0%{mvncsdk2_support}
 BuildRequires:	pkgconfig(libmvnc)
 %endif
+
+# for Vivante
+%if 0%{?vivante_support}
+BuildRequires:  pkgconfig(opencv)
+BuildRequires:  pkgconfig(ovxlib)
+BuildRequires:  pkgconfig(amlogic-vsi-npu-sdk)
+BuildRequires:  pkgconfig(dann)
+Requires:       gst-plugins-good
+Requires:       gst-plugins-good-extra
+Requires:       gst-libav
```

Comment:
LINE 131-133 appear incorrect.

Reply:
The three packages are required for the Vivante-based example applications (e.g., custom reference boards) by default.

Reply:
Goto Line 228

Reply:
Line 131-133 is moved to Line 228.